docs(screens): Add docs for the screens module (#8498)
getsentry/sentry-docs: src/docs/product/performance/mobile-vitals.mdx

- [Android](/platforms/android/performance/instrumentation/automatic-instrumentation/#time-to-full-display)
- [Apple](/platforms/apple/performance/instrumentation/automatic-instrumentation/#time-to-full-display)

## Screens

The **Screens** page shows an overview of how long it takes for your application to load its screens. Here you can identify slow or regressed screens and get additional information to understand the factors contributing to the slowness of both time to initial display (TTID) and time to full display (TTFD).

> While Sentry automatically tracks TTID, [TTFD](#time-to-initial-display-and-time-to-full-display) requires for the `reportFullyDisplayed()` API to be called manually to report that the screen has loaded all of its content and is fully displayed.

Reviewer: ```suggestion
While Sentry automatically tracks TTID, [TTFD](#time-to-initial-display-and-time-to-full-display) requires that the `reportFullyDisplayed()` API be called manually to report that the screen has loaded all of its content and is fully displayed.
```

> **Minimum SDK Requirements:**
> 
> For Android:
> 
> - `>=5.0` for automatic activity transactions
> - `>=6.10.0` for TTID
> - `>=6.17.0` for TTFD, released March 2023
> 
> iOS:
> 
> - `>=3.13` for UIViewController transactions
> - `>=8.4.0` for TTID+TTFD, released April 2023

Reviewer: ```suggestion
### Minimum SDK Requirements:

**For Android:**

- `>=5.0` for automatic activity transactions
- `>=6.10.0` for TTID
- `>=6.17.0` for TTFD, (released March, 2023)

**For iOS:**

- `>=3.13` for UIViewController transactions
- `>=8.4.0` for TTID+TTFD, (released April, 2023)
```

By default, the **Screens** page displays metrics for the two releases with the highest screen counts for the time range you’ve selected. To choose a different set of releases to compare, use the “release selector” at the top of the page.

The charts section shows a comparison of TTID and TTFD for your top five screens across your two selected releases.

The sortable table below shows a list of your application's screens with the average TTID and TTFD for each release.

**Reasons Why You Might Not Be Seeing Any Data:**

- You don’t have any transactions with op `ui.load`
- Your SDKs don’t meet the minimum SDK requirements

> ## Screen Summary
> 
> To get additional information about any of your application's screens, click on them to get to the **Screen Summary** page. Here, you’ll see a breakdown of TTID and TTFD values broken out by [device class](/product/reference/search/searchable-properties/#device-classification) (high, medium, low, or unknown), across your selected releases. This will help you understand how users with different device performance levels are being affected.
> 

Reviewer: ```suggestion
## Screen Details Page

To get additional information about any of your application's screens, click on them to get to the **Screen Details** page. Here, you’ll see a breakdown of TTID and TTFD values broken out by [device class](/product/reference/search/searchable-properties/#device-classification) (high, medium, low, or unknown), across your selected releases. This will help you understand how users with different device performance levels are being affected.

```

The Screen Details page also shows the average TTID, average TTFD, and Count (the number of times a screen has been called) - broken out by release.

> To quickly get to slow TTID and TTFD events and profiles for each release broken out by specific device class, use the “Device Class” dropdown at the top of the table.
> 

Reviewer: ```suggestion
To see slow TTID and TTFD events and profiles for each release broken out by specific device class, use the “Device Class” filter at the top of the table.

```

At the bottom of the page, you’ll see a table of spans that occurred during screen load, sorted by the most time your screen spent in that span. Here, you’ll be able to:

- Spot span regressions by comparing the average duration of each span across your two releases of choice.

- Find the slowest spans in the release you’re interested in by looking at average span duration.

- See which spans occur most frequently by looking at “Count”. (This might help you identify excessive io calls or repeated work that might be occurring during screen load.)

> Eligible span operations:

Reviewer: ```suggestion
**Eligible span operations:**
```
